src/main/java/li/cil/oc2/common/entity/Robot.java

```diff
@@ -866,6 +867,17 @@ public ItemStack getStackInSlot(@Parameter("slot") final int slot) {
             return inventory.getStackInSlot(slot);
         }
 
+        @Callback(synchronize = false)
+        public String inspect(@Parameter("side") @Nullable final RobotOperationSide side) {
+            if (side == null) throw new IllegalArgumentException();
+            final Level level = Robot.this.level;
+            if (!(level instanceof final ServerLevel serverLevel)) {
+                throw new IllegalArgumentException();
+            }
+            final BlockPos pos = Robot.this.blockPosition();
+            return level.getBlockState(pos.relative(RobotOperationSide.toGlobal(Robot.this, side))).getBlock().getDescriptionId();
```

Comment:
Do we maybe want to explicitly check if the block is "air" and return null instead if that's the case?